### ISSUE TITLE
CSP Issue after 2.1.0 update

### ISSUE THREAD

**Issue by dozzed** [craftcms/commerce-stripe#74]
After updating to 2.1.1 JS stops working ONLY on pages containing external Stripe JS. Chrome doesn't show a specific error only traditional JS errors: 

`Uncaught TypeError: $(...).Lazy is not a function`, etc

Safari however shows this on the console:
![image](https://user-images.githubusercontent.com/6993471/63099559-98f85080-bf75-11e9-8f74-e19af2a905bd.png)

I've used various configurations for `http-equiv="Content-Security-Policy" content="script-src * 'unsafe-inline';"` to allow external scripts but no luck.

Production sites are running Stripe 2.0.1.2 without any issues. Nothing was added or removed from the checkout process. 

Has anyone encountered this issue before? Not really sure what to do.

Cheers

**Comment by andris-sevcenko** [comment]
Can you try switching to HTTPS?

**Comment by dozzed** [comment]
I just implemented https on my local environment but the issue is still present. All projects running 2.0.1.2 or earlier are fine (even from non https environments).

**Comment by andris-sevcenko** [comment]
Here's what's changed since 2.0.1.2: https://github.com/craftcms/commerce-stripe/compare/2.0.1.2...master

It still looks like an environmental issue.

**Comment by dozzed** [comment]
After banging my head for two days I finally found the solution. As it usually happens it was a rather simple fix. I just moved my JS to the end of body (which should have been there anyways). Now everything is working again.

I'm thinking this has to do with a recent Craft CMS update (3.2.5 or later) and not the Stripe plugin but I can't pinpoint the exact cause right now.

Anyway having JS on the head of the body was causing issues with Stripe external sources.


Appreciate the responses.
